kenwood/elecraft.c

```diff
@@ -96,11 +96,6 @@ int elecraft_open(RIG *rig)
     rig_debug(RIG_DEBUG_VERBOSE, "%s called, rig version=%s\n", __func__,
               rig->caps->version);
 
-    if (!rig)
-    {
-        return -RIG_EINVAL;
-    }
-
```

Comment:
I know cppchecker complained about these tests.  They've been in there as long as I can recall and maybe they offer no protection and are simply redundant.  I suppose my ignorance/forgetfulness is showing!